index.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener("DOMContentLoaded", () => {
+    axios.get("http://localhost:3000/").then(res => {
+    res.data.forEach(person => {
+        let container = document.querySelector('#container')
+        let personDiv = document.createElement('div')
+         let personName = person.name.first + " " + person.name.last + " " + person.gender
```

Comment:
Try using string interpolation, its much easier to do that the way you have it 

nomal way : `person.name.first + " " + person.name.last + " " + person.gender`
string interpolation : ` `${person.name.first} ${person.name.last} ${person.gender}` `